bundles/org.openhab.binding.energidataservice/src/main/java/org/openhab/binding/energidataservice/internal/ApiController.java

```diff
@@ -110,7 +112,7 @@ public ElspotpriceRecord[] getSpotPrices(String priceArea, Currency currency, Da
         }
 
         Request request = httpClient.newRequest(ENDPOINT + DATASET_PATH + DATASET_NAME_SPOT_PRICES)
-                .param("start", start.toString()) //
+                .timeout(REQUEST_TIMEOUT_MS, TimeUnit.MICROSECONDS).param("start", start.toString()) //
```

Comment:
```suggestion
                .timeout(REQUEST_TIMEOUT_MS, TimeUnit.MILLISECONDS) //
                .param("start", start.toString()) //
```

(or seconds, see other comment).

I would need to check average response time over some time to know if this is a reasonable timeout for this cloud service binding.

Usually the service is called only once per day at a very specific time. There is a retry policy with exponential backoff, so we'll retry automatically when hitting the timeout. However, it would be much preferable get it on first call to have the energy prices updated as fast as possible. Also, timeout is not part of the policy, so if it takes longer than 10 seconds, it might also take longer on next call etc.

I'm not saying it's bad, just that there are some considerations to make. I'll see if I can setup some metrics to get to know the min/avg/max response times.

Reply:
Let me know and i will adjust it. If at some point this PR is waiting for this, i can set it to 30seconds for now. And you might adjust it in a seperate PR.

Reply:
Sure, 30 seconds would make me more calm and give me some time to measure and tweak.

Reply:
Done